app/Http/Controllers/People/ActivitiesController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Http\Controllers\People;
+
+use App\Contact;
+use App\Activity;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\People\ActivitiesRequest;
+
+class ActivitiesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param Contact $contact
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Contact $contact)
+    {
+        return view('people.activities.index')
+            ->withContact($contact);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @param Contact $contact
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Contact $contact)
+    {
+        return view('people.activities.add')
+            ->withContact($contact)
```

Comment:
I'm pretty sure it's a laravel thing, but I've never seen it covered in the documentation.